src/canopy/models/data_models.py

```diff
@@ -112,3 +112,11 @@ class SystemMessage(MessageBase):
 class AssistantMessage(MessageBase):
     role: Literal[Role.ASSISTANT] = Role.ASSISTANT
     content: str
+
+
+class OpenAIClientParams(BaseModel):
```

Comment:
This class definitely doesn't belong here.  
It's an inner config-level detail of a specific LLM (theoretically out of many supported LLMs). The `data_models` file is for our main API data classes like `Query` and `Context` etc.  

I think you can safely move it inside the `llm/openai.py` file.

Reply:
removed